test/blackbox-tests/test-cases/watching/dune

```diff
@@ -21,5 +21,6 @@
 
 (cram
  (applies_to watching-eager-concurrent-build-command
-  watching-eager-concurrent-exec-command)
```

Comment:
Does this get stuck? Since we are using passive watch mode I wouldn't expect this to be the case.